docs/essentials/feature-experimentation.md

```diff
@@ -85,4 +85,42 @@ Statistical significance has been achieved if this number is either 0% or 100%.
 ![Positive Metric Results](/feature-experiment-positive-results.png)
 
 **Negative Results**
-![Negative Metric Results](/feature-experiment-negative-results.png)
\ No newline at end of file
+![Negative Metric Results](/feature-experiment-negative-results.png)
+
+
+## Experimentation using a Custom Property for Randomization
+
+:::info
+For documentation on this functionality outside of the context of experimentation you can check out our documentation dedicated to this topic [here](/extras/advanced-targeting/randomize-using-custom-property).
+:::
+
+DevCycle typically uses the User ID as the primary key for Targeting Rules, Feature rollouts and randomization. However, in certain scenarios, Features you release are intended to be rolled out to a cohort of users vs an individual user. For example, a new feature in a B2B platform might impact an entire organization rather than a single user within that organization. In such cases, you can randomize and rollout by using a Custom Property. 
+
+### What are Experiments that Randomize Using a Custom Property?
+When running an experiment where you randomize using a Custom Property, the experiment is applied to a set of users (those who possess a Custom Property) rather than individual users. This means that every user who has that Custom Property will experience the same Feature Variation, such as being part of the control or the test variant. This approach allows you to assess the impact of changes on the group as a whole.
+
+Groups in DevCycle are defined using Custom Properties. These groups could be companies, tenants, geographic locations, or any set of users sharing common characteristics.
+
+![Randomization Grouping](/custom-property-groups.png)
+
+### How to Randomize Using a Custom Property in Experiments
+
+To set this up, create a Targeting Rule that serves a Random Distribution of the Variations. 
+
+When you select `Random Distribution`, a `Randomize Using` field will appear at the bottom of the Targeting Rule under the `Schedule` section. The dropdown will populate with all existing Custom Properties. Select the Custom Property you wish to use for your random distribution. If you are both randomizing distribution and using a gradual rollout of some form, the Custom Property will be used for both forms of randomization, keeping distribution sticky based off of that property.
```

Comment:
can use `the` instead of `a` or remove both and just keep it as `Randomize Using` field will appear ...
```suggestion
When you select `Random Distribution`, `Randomize Using` field will appear at the bottom of the Targeting Rule under the `Schedule` section. The dropdown will populate with all existing Custom Properties. Select the Custom Property you wish to use for your random distribution. If you are both randomizing distribution and using a gradual rollout of some form, the Custom Property will be used for both forms of randomization, keeping distribution sticky based off of that property.
```